plugins/lookup/passwordstore.py

```diff
@@ -304,6 +342,6 @@ def run(self, terms, variables, **kwargs):
                 else:
                     result.append(self.get_passresult())
             else:                     # password does not exist
-                if self.paramvals['create']:
+                if self.paramvals['missing'] == 'create':
```

Comment:
What will plugin do if `create` is `true` and `missing` has been set to `error` with this change? Is this changing the current behaviour? Should it be like this?
```suggestion
                if self.paramvals['create'] or self.paramvals['missing'] == 'create':
```

Reply:
See my comment below.

Reply:
As stated in description, creates takes precedence. If you use `create=true` it will impose `missing=create`. So the behavior is backward compatible.